sample.env

```diff
@@ -105,8 +105,10 @@ MAPBOX_INIT_ZOOM="10"
 
 # Webapp themes: light, dark, cdotDark
 # WEBAPP_THEME_LIGHT value is used if the browser is in light mode. If the browser is in dark mode, WEBAPP_THEME_DARK is used
-WEBAPP_THEME_LIGHT="dark" # if not set, defaults to 'light'
-# WEBAPP_THEME_DARK="dark" # if not set, defaults to 'dark'
+# if not set, defaults to 'light'
+WEBAPP_THEME_LIGHT="cdotDark"

```

Comment:
Did you say this actually needs to be REACT_APP_WEBAPP_THEME_LIGHT to get it to pop?

Reply:
That is only for the npm-specific .env file (.env.local)